docs/learn-governance.md

```diff
@@ -181,15 +183,17 @@ JJ: Votes `No` for a 16 week lock period => 150 * 3 = 450 Votes
 
 approve = 600
 against = 450
-voters = 1050
+turnout = 750
 electorate = 1500
 ```
 
-![\Large \frac{450}{\sqrt{1050}}&space;<&space;\frac{600}{\sqrt{1500}}](https://latex.codecogs.com/svg.latex?\large&space;\frac{450}{\sqrt{1050}}&space;<&space;\frac{600}{\sqrt{1500}})
+![\Large \frac{450}{\sqrt{750}}&space;<&space;\frac{600}{\sqrt{1500}}](https://latex.codecogs.com/svg.latex?\large&space;\frac{450}{\sqrt{750}}&space;<&space;\frac{600}{\sqrt{1500}})
 
-![\Large {13.887}&space;<&space;{15.492}](https://latex.codecogs.com/svg.latex?\large&space;{13.887}&space;<&space;{15.492})
+![\Large {16.432}&space;<&space;{15.492}](https://latex.codecogs.com/svg.latex?\large&space;{16.432}&space;<&space;{15.492})
 
-Based on the above result, the proposal will be approved. In addition, only the winning voter's
+Since the above example is a public proposal, `Super-Majority Approve` would be used to calculate the result. 
+And `Super-Majority Approve` requires more `aye` votes to pass the proposal when turnout is low, therefore, 
+based on the above result, the proposal will be rejected. In addition, only the winning voter's
```

Comment:
```suggestion
Since the above example is a public referendum, `Super-Majority Approve` would be used to calculate the result. 
And `Super-Majority Approve` requires more `aye` votes to pass the referendum when turnout is low, therefore, 
based on the above result, the referendum will be rejected. In addition, only the winning voter's
```